cekit/template_helper.py

```diff
@@ -111,7 +113,9 @@ def package_manager_flags(self, pkg_mgr):
             # However this works at the moment...
             #
             return "update && apt-get --no-install-recommends"
-        elif "microdnf" in pkg_mgr:
+
+        default = "--setopt=tsflags=nodocs"
+        if "microdnf" in pkg_mgr:
```

Comment:
not a Python developer but as far as I understood if no manager flags are specified and the manager is microdnf, the `--setopt=install_weak_deps=0` is set by default ... but it's exactly what brakes backward compatibility related to #714, or? What am I missing?

Reply:
This is still to be debated with @jmtd and @goldmann but this PR allows user's who are on latest release and using ubi8 to have the preferred options by default, with those needing to have backwards compatibility can override as desired.

Reply:
tbh, I think that the idea of backward compatibility is doing nothing for people using older stuff not the need to override something to make it working again. The newer cekit version is breaking backward compatibility actually.